cypress/e2e/API/api_EngageSphere.cy.js

```diff
@@ -0,0 +1,160 @@
+/// <reference types="cypress" />
+
+//DÚVIDAS!!!
+//to.be.at.greaterThan(9999).and.to.be.lessThan(50000)
+//Retorna que o customer Littel  tem 10.000 funcionários e também que ele possuí menos de 50.000
+
+describe('ENGAGE SPHERE API TESTING', () => {
+
+    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
+
+    it('SUCCESSFULLY RETRIEVES CUSTOMERS (E.G., CHECKS FOR THE 200 STATUS CODE)', () => {
+        cy.request('GET', CUSTOMERS_API_URL + '/customers')
+            .then(({ body, status }) => {
+                expect(status).to.be.eq(200);
+                expect(body.pageInfo.totalCustomers).to.be.is.not.null;
```

Comment:
Que tal uma verificação mais simples, tal como a seguinte?

```suggestion
                expect(body.pageInfo.totalCustomers).to.exist;
```